src/components/ComboBox/ComboBox.stories.tsx

```diff
@@ -50,17 +52,26 @@ const defaultItems = [
     label: 'アイテムのラベルが長い場合（ダミーテキストダミーテキストダミーテキストダミーテキスト）',
     value: 'value-6',
   },
+  {
+    label: (
+      <>
+        <Stack as="span">アイテムのラベルがReactNodeの場合</Stack>
+        <Stack as="span">（ダミーテキストダミーテキストダミーテキストダミーテキスト）</Stack>
+      </>
```

Comment:
挙動としては問題ないんですが、Stack が内部的に `display: flex;` となり Stack しているだけなので、意図的には次の提案のようなコードが良さそうです。

```suggestion
      <Stack as="span" gap={0.25}>
        <span>アイテムのラベルがReactNodeの場合</span>
        <span>（ダミーテキストダミーテキストダミーテキストダミーテキスト）</span>
      </>
```

Reply:
Stackをstackされる側とユースケースを勘違いしてました 🙏
 こちらも修正済みです！
https://github.com/kufu/smarthr-ui/pull/3270/commits/bdfea4548e12df77ac8b6efaa12bc79c075dc03e

Reply:
良かったです!